pkg/stub/handler.go

```diff
@@ -589,7 +638,8 @@ func (h *Handler) Handle(event util.Event) error {
 		// based on the current conditions of the Config.
 		cfg = h.refetchCfgMinimizeConflicts(cfg)
 		//TODO remove this setting of version once we start getting samples for z or ppc
-		if util.IsNonX86Arch(cfg) {
+		// and registry.redhat.io starts support IPv6
+		if util.IsNonX86Arch(cfg) || util.IPv6Only() {
```

Comment:
If we're removed, do we reach this code path? If so then we don't need to check for IPv6 here.

Reply:
bootstrapping as removed has some implications wrt forcing the necessary clusteroperator bits to appease the CVO during initial install, like the version here.

But maybe  we could do that during the initial create and remove this.  I'll look into it.